st2client/st2client/models/core.py

```diff
@@ -376,6 +376,31 @@ def delete(self, instance, **kwargs):
 
         return True
 
+    @add_auth_token_to_kwargs_from_env
+    def clone(
+        self,
+        source_ref,
+        dest_pack,
+        dest_action,
+        overwrite,
+        **kwargs,
+    ):
+        url = "/%s/%s/clone/?overwrite=%s" % (
+            self.resource.get_url_path_name(),
+            source_ref,
+            overwrite,
```

Comment:
Overwrite is already in the JSON payload. Is this duplicate? Let's just use the overwrite in the payload below.